src/CommunityToolkit/Xamarin.CommunityToolkit/PlatformConfiguration/Multiplatform/StatusBar/StatusBar.shared.cs

```diff
@@ -0,0 +1,53 @@
+﻿using System;
+using Xamarin.Forms;
+
+namespace Xamarin.CommunityToolkit.PlatformConfiguration.Multiplatform
+{
+	public static partial class StatusBar
+	{
+		/// <summary>
+		/// Sets the color of application's status bar
+		/// </summary>
+		public static readonly BindableProperty ColorProperty = BindableProperty.Create(
+			"Color", typeof(Color), typeof(StatusBar), Color.Default, propertyChanged: (_, _, newValue) => SetColor((Color)newValue));
+
+		/// <summary>
+		/// Sets style of application's status bar
+		/// </summary>
+		public static readonly BindableProperty StyleProperty = BindableProperty.Create(
+			"Style", typeof(StatusBarStyle), typeof(StatusBar), StatusBarStyle.Default, propertyChanged: (_, _, newValue) => SetStyle((StatusBarStyle)newValue));
```

Comment:
We don't use an inline delegate, can you extract it to a method?

Reply:
Is there some downsides of using lambdas here? I just don't want to create two private methods that are only used to call another method with one of the arguments, and which are only used once.

Reply:
A method is better to debug than a lambda, and the IDE will do the hard for you in the refactoring.